src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
The class implementation should not begin with an empty line. Please remove the unnecessary empty lines at the start of the class definition.